subsystems/intakesubsystem.py

```diff
@@ -10,15 +11,22 @@ def __init__(self) -> None:
         self.intakeDeployed = False  # default to intake retracted and not reversed
         self.intakeReversed = False
 
-        self.pneumaticsHub = PneumaticHub(1)
+        self.pneumaticsHub = PneumaticHub(constants.kPneumaticsHubCanID)
 
-        self.intakeSolenoid = Solenoid(PneumaticsModuleType.REVPH, 0)
+        self.intakeSolenoid = Solenoid(
+            PneumaticsModuleType.REVPH, constants.kIntakeSolenoidChannelId
+        )
         self.intakeSolenoid.set(False)
 
-        if RobotBase.isReal():
-            pass
-        else:
-            self.intakeMotor = PWMVictorSPX(constants.kSimIntakeMotorPort)
+        self.intakeMotor = Falcon(
+            constants.kIntakeMotorName,
+            constants.kIntakeMotorId,
+            constants.kSimIntakeMotorPort,
+        )
+        # if RobotBase.isReal():
```

Comment:
Can this commented out code be removed?

Reply:
That was there because the Falcon metaclass is currently not operational on the real robot. I was using that to test the motors (make sure it was a problem with the class, not something else). We can remove it if we're sure we don't need the testing anymore.

Reply:
would rather just fix falcon than commit dead code

Reply:
Sure feel free, just last I knew we had no idea what was wrong so I wasn't sure how long we would be keeping the test code lol